iterations/0.md

```diff
@@ -159,11 +159,11 @@ If it’s clear that the project is viable (for example, if you’re using only
 
 Determine whether the project you’re proposing is at the right scale.
 
-This is subjective and difficult to estimate, so you must talk to the staff about it. At the bare minimum your project must include one _non-data-manipulation_ feature; ideally it includes several.
+This is subjective and difficult to estimate, so you must talk to the staff about it. At the bare minimum your project must include one _non-data-manipulation (non-CRUD)_ feature; ideally it includes several.
```

Comment:
CRUD is jargon. Either we introduce and explain it, or we don’t use it. I’d prefer not to use it.

Reply:
Changed

Reply:
Did you push? I still see the change…

Reply:
I just pushed with the changes we talked about in these comments. I'm still writing up the importance of testing for each feature.